queries.sql

```diff
@@ -181,4 +181,12 @@ LEFT JOIN animals ON animals.id_animals = visits.id_animal
 LEFT JOIN species ON animals.id_species = species.id_species
 LEFT JOIN vets ON vets.id_vets = visits.id_vet
 WHERE vets.name_vets = 'Maisy Smith'
-GROUP BY species.name;
\ No newline at end of file
+GROUP BY species.name;
+
+CREATE INDEX idx_visits_vet_id ON visits(id_vet);
+CREATE INDEX ON visits (id_animal);
+CREATE INDEX ON owners ((lower(email)));
```

Comment:
Please add these queries (add indexes to foreign keys) to `schema.sql` file.

Reply:
Good to know, thanks for noticing it I just forgot to move them into `schema.sql`